benchmarks/shelley3pools/configuration-generator.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+. ./configuration/parameters
+
+NNODES=${NNODES:-3}
+
+for i in $(seq 1 ${NNODES}) 
+do
+    cp ./configuration/configuration-template.yaml ./configuration/configuration-node-${i}.yaml
```

Comment:
I think we should place those in a specific directory which we can then delete and recreate.

Reply:
I'll put them into `./tmp`.